.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,19 +5,26 @@ for guidance on how to set up your development environment,
 run the test suite, write new integrations, and more.
 -->
 
-**What does this PR do?**
+**What does this PR do?**:
```

Comment:
Grammar nitpick: is having a colon after a question mark correct? Should the title be re-phrased `What this PR does: `

Reply:
:+1: Fixed in 6ae39459d338bef1b9d936aaf471bca75db4f235